errors/errors.go

```diff
@@ -0,0 +1,21 @@
+package errors
+
+import (
+	"os"
+)
+
+// Code exit code
+type Code int
+
+const (
+	// CodeParseRequestLength error parsing request length
+	CodeParseRequestLength Code = iota + 10
```

Comment:
What's the rationale behind the magic number `10` here?

Reply:
I got an idea to not only send this error code to the browser, but also to use this same code as exit code here in the app for consistency - after all, if the app is sending an error, there's nothing left for it to do anyway, so it should exit with some exit code.

However, starting from `1` here doesn't work very well - the app can crash, the app can exit with `log.Fatal`, and those cases also generate exit code `1`. So I thought, what if we start counting our own custom exit codes from a larger number, which will not collide with crashes or accidental crashes. My options were `2` or `10` 🙂 

What do you think about this whole idea?

Reply:
> I got an idea to not only send this error code to the browser, but also to use this same code as exit code here in the app for consistency...

I really like this - good idea!

> However, starting from 1 here doesn't work very well - the app can crash, the app can exit with log.Fatal, and those cases also generate exit code 1. So I thought, what if we start counting our own custom exit codes from a larger number, which will not collide with crashes or accidental crashes. My options were 2 or 10.

That makes sense :-). `10` works, and will be well away from everything else.

It would be useful if you could add a comment there that explains it, and also stops people reordering or inserting values though - we don't want those error codes changing if somebody decides to edit this, and doesn't understand what you've done.